### ISSUE TITLE
justReturn?

### ISSUE THREAD

**Issue by tyynetyyne** [vishesh/whalebin#17]
Something funny happening here: http://bigbang.ccs.neu.edu/get/dmPy6
Paste is showing just a lot of this:
justReturn@http://bigbang.ccs.neu.edu/get-full/dmPy6:13512:16
_62007@http://bigbang.ccs.neu.edu/get-full/dmPy6:16649:8
...

**Comment by vishesh** [comment]
Yeah, we get this stack trace in some cases. Mostly in Firefox. Try this on Chrome. Meanwhile I will try to reproduce and file a bug in Whalesong.

**Comment by tyynetyyne** [comment]
Well, I tried this in Chrome and there I didn't get the justReturn error but the animation didn't work as expected: it flashed the first image (back rectangle) on the lower right corner. Then it started to draw the spiral outside of the animation area and then it just falled apart... Here is the same code running in WeScheme so you can see how it show behave: http://www.wescheme.org/view?publicId=t44z4PS9Vg
